logging.go

```diff
@@ -33,23 +45,23 @@ const LOGS = "logs"
 // LogIncomingRequests - Logs request traffic into our app
 func LogIncomingRequests(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		path := filepath.Join(LOGS, CONNECTOR)
-		writer, err := GetLogWriter(path)
-		if err != nil {
-			logrus.Error(err.Error())
+		if !config.Test {
```

Comment:
What does this do? Both branches seem to do the same thing.

Reply:
They do, the plan was to disable stderr if testing, can change it